transformations/workspace/index.md

```diff
@@ -134,6 +137,27 @@ or [File Output Mapping](/transformations/mappings/#file-output-mapping) (or bot
 Unloading data is useful, for example, when your ad-hoc analysis leads to
 valuable results, or when you trained a new model which you'd like to use in transformations.
 
+### Shared Workspaces
+
+Every member of a project can share his workspace with all project members. If you share workspace
+with other project members, then every member of a project can use your workspace, load data to it,
+restore it, etc.
```

Comment:
k tomu bych ale napsal neco v tom stylu, ze je to porad pod jednim uzivatelem, aby se lidi nedivili, az si budou prepisovat veci navzajem